docs/src/contribute/development-environment.md

```diff
@@ -7,6 +7,8 @@ eleventyNavigation:
     order: 6
 ---
 
+{%- from 'components/npm_tabs.macro.html' import npm_tabs %}
+
```

Comment:
```suggestion
{%- from 'components/npm_tabs.macro.html' import npm_tabs with context %}

```

this way we can use global data in macro components.

Can you please add the `with context` key word in the end of each import of `npm_tabs` as above in all the files?